app/controllers/accounts_controller.rb

```diff
@@ -12,7 +12,7 @@ def index
   def create
     authorize :execute
 
-    api_key = Account.create account_name
+    api_key = Account.create account_name, params[:owner_id]
```

Comment:
I think this should be `:owner` for consistency. It also probably needs a default (`current_user`?) here; in fact, I'm not sure it should be configurable at all.

Reply:
Oh that's true, I think using the current user would actually work quite nicely. I'll do that.